frontend/src/components/filtertable.js

```diff
@@ -32,17 +32,13 @@ export class FilterTable extends React.Component {
     rows: PropTypes.array,
     actions: PropTypes.array,
     filters: PropTypes.array,
-    activeFilters: PropTypes.object,
-    hideFilters: PropTypes.array,
```

Comment:
We use this table in a few places, and I'm a bit worried about removing this stuff without proper testing. Did you check the runs and results pages?

Reply:
I'm going to check this again tomorrow (Friday) to see how it affects the other tables.

Reply:
Ok I need to refine some things.

Reply:
As you can see below I thought I could put this back into draft, but I guess not lol. Anyways, I removed all of those changes to FilterTable, so there should be no worries. I checked other pages and they looked good. Just had to modify some stuff on my view. You can get the same functionality I was after if you pass those functions to MetaFilter and not worry about it in FilterTable.